applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_FIC_element.hpp

```diff
@@ -105,16 +105,21 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) UPwSmallStrainFICElement
     std::string Info() const override
     {
         std::stringstream buffer;
+        std::stringstream claw_buffer;
+        if (mConstitutiveLawVector.size() != 0) {
+            claw_buffer << mConstitutiveLawVector[0]->Info();
+        } else {
+            claw_buffer << "not defined";
+        }
         buffer << "U-Pw smal strain FIC Element #" << this->Id()
-               << "\nConstitutive law: " << mConstitutiveLawVector[0]->Info();
+               << "\nConstitutive law: " << claw_buffer.str();
         return buffer.str();
     }
 
     // Print information about this object.
     void PrintInfo(std::ostream& rOStream) const override
     {
-        rOStream << "U-Pw smal strain FIC Element #" << this->Id()
-                 << "\nConstitutive law: " << mConstitutiveLawVector[0]->Info();
+        rOStream << Info();
```

Comment:
Nice that we re-use the Info() function here! Still better, could we add this as default behavior to the Element? This would allow us to remove all these PrintInfo overrides with very generic behavior.

Reply:
Good point it might not be as easy as it sounds. We could try it once one a branch and see how many issues arise to get a sense of the scope of the work this would bring?